client/src/components/header/Header.js

```diff
@@ -1,15 +1,21 @@
 import React from 'react';
 import './Header.css';
-import Button from '@mui/material/Button';
-import Stack from '@mui/material/Stack';
+import {useDispatch, useSelector} from "react-redux";
+import {logoutServ} from "../../service/auth.service";
+
 
 const Header = () => {
+    const nickname = useSelector(state => state.user)
+    const isLoggedIn = useSelector(state => state.isLoggedIn)
+    const dispatch = useDispatch()
     return (
         <div className="App-header">
             <div className="Nav">
                 <a href="/"><img src="images/carbuy.png" alt="logo" /></a>
                 <a href="/" className="Plca"><button className="Plc" >Place an advertisement</button></a>
-                <a href="/"><button className="Login" >Log In</button></a>
+                {isLoggedIn ?
+                    <button className="Login" id="Log" onClick={() => dispatch(logoutServ())}>Hello, {nickname}! Log Out?</button>
```

Comment:
() => dispatch(logoutServ()) Do it in markup is not good idea, create function and call it